sdk/cosmos/azure_data_cosmos/src/partition_key.rs

```diff
@@ -95,6 +77,19 @@ impl<T: Into<PartitionKey>> From<T> for QueryPartitionStrategy {
 #[derive(Debug, Clone)]
 pub struct PartitionKey(Vec<PartitionKeyValue>);
 
+impl PartitionKey {
+    /// A single null partition key value, which can be used as the sole partition key or as part of a hierarchical partition key.
+    pub const NULL: PartitionKeyValue = PartitionKeyValue(InnerPartitionKeyValue::Null);
+
+    /// An empty list of partition key values, which is used to signal a cross-partition query, when querying a container.
+    pub const EMPTY: PartitionKey = PartitionKey(Vec::new());
+
+    #[allow(dead_code)] // Used in the query_engine feature.
+    pub(crate) fn is_empty(&self) -> bool {
+        self.0.is_empty()
```

Comment:
Presumably not, but just to ask the question since I'm not an expert in Cosmos: should this also return `true` for `EMPTY`?

Reply:
This should indeed return true for `EMPTY`. `assert!(PartitionKey::EMPTY.is_empty())` would not panic.